tests/spec/fs.stat.spec.js

```diff
@@ -92,3 +92,32 @@ describe('fs.stat', function() {
     });
   });
 });
+
+describe('fs.promises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat).to.be.a('function');
+  });
+  
+  it('should return a stat object if file exists', function(done){
```

Comment:
To make this test more promise-friendly, please remove the `done` from the callback function arguments.